microservice/subscription/src/main/java/com/cumulocity/microservice/subscription/repository/impl/CurrentMicroserviceRepository.java

```diff
@@ -72,6 +72,8 @@ public Iterable<ApplicationUserRepresentation> getSubscriptions() {
         try {
             return rest().get(url, APPLICATION_USER_COLLECTION_MEDIA_TYPE, ApplicationUserCollectionRepresentation.class);
         } catch (final Exception ex) {
+            RestOperations rest = rest();
```

Comment:
I might overlook something but I don't understand what the method call should do here? Is this just a left-over or does it has a specific purpose?